modules/algebrasAndModel/src/main/scala/subscriptionAlgebra/SubscriptionServiceAlgebra.scala

```diff
@@ -0,0 +1,9 @@
+package subscriptionAlgebra
+
+import utils.Types.{GetSubscriptionData, Id, PostSubscriptions}
+
+trait SubscriptionServiceAlgebra[F[_]] {
+  def get(id: Id): F[Option[GetSubscriptionData]]
```

Comment:
I just figured out that this should probably be the `slackId` because the ID being passed to the the Url is a String.

Reply:
Yes, you'd probably want to have a relation with the one-to-one mapping between your internal ids and the upstream services like Slack. Let me know if you need help with this.

Reply:
Yeah I will definitely reach out yo You on this one. Thanks @moleike